### ISSUE TITLE
Trial Mode after update

### ISSUE THREAD

**Issue by koopa** [ONLYOFFICE/CommunityServer#396]
A few days ago we updated onlyoffice:
https://github.com/ONLYOFFICE/CommunityServer/issues/394

Now suddenly our installation is in trial mode:
![grafik](https://user-images.githubusercontent.com/195897/173336490-4f3d727a-549e-44f9-8a5e-206e38e8db96.png)

And when I click this icon, it tries to go to `/controlpanel/activate` but then the server returns a white page. In the raw response I then saw:
```
/controlpanel/activate 
402 Payment Required
```
What does this mean?

**Comment by koopa** [comment]
Just to be clear, we are using GROUPS and before the update it was not in any trial mode. How can it be re-activated?

**Comment by Carazyda** [comment]
Hello @koopa it looks like you upgraded to the enterprise version because you did not specify a parameter `-it` for the script. Now, in order for you to downgrade back to the free version, you need to uninstall the Enterprise Document Server `sudo apt-get remove onlyoffice-documentserver-ee`, and install the free one if you're using the workspace `sudo apt-get install onlyoffice-documentserver`. If you are using groups, you do not need to install Document Server. Then clear the value of key `license.file.path` in the configs `/var/www/onlyoffice/WebStudio/web.appsettings.config` and `/var/www/onlyoffice/Services/TeamLabSvc/TeamLabSvc.exe.config` , and clear the table tenants_tariff in the database `truncate tenants_tariff;`. And restart all services or the entire server.

**Comment by koopa** [comment]
Thanks for the instructions @Carazyda!
So next time I need to update I would then run `./install-Debian.sh -it GROUPS -u true`, correct?
Also maybe a stupid question, but how would I just restart all the services without rebooting?

**Comment by Carazyda** [comment]
> So next time I need to update I would then run ./install-Debian.sh -it GROUPS -u true, correct?
Yes, if you use Onlyoffice Groups. If Workspace - change it to `-it WORKSPACE`.

> Edit: I tried systemctl restart onlyoffice* but it was not enough it seems.
You also need to restart our web site - `systemctl restart monoserve`.

I'm very glad that your problem was solved.
